unicodetools/data/ucd/dev/DerivedCoreProperties.txt

```diff
@@ -6729,6 +6730,7 @@ FFDA..FFDC    ; ID_Start # Lo   [3] HALFWIDTH HANGUL LETTER EU..HALFWIDTH HANGUL
 10E80..10EA9  ; ID_Start # Lo  [42] YEZIDI LETTER ELIF..YEZIDI LETTER ET
 10EB0..10EB1  ; ID_Start # Lo   [2] YEZIDI LETTER LAM WITH DOT ABOVE..YEZIDI LETTER YOT WITH CIRCUMFLEX ABOVE
 10EC2..10EC4  ; ID_Start # Lo   [3] ARABIC LETTER DAL WITH TWO DOTS VERTICALLY BELOW..ARABIC LETTER KAF WITH TWO DOTS VERTICALLY BELOW
+10EC6         ; ID_Start # Lo       ARABIC LETTER THIN NOON
```

Comment:
FYI -- ID_Start seems a bit odd for a letter that is only attested in medial form, but I think we don't have a mechanism for exceptionally keeping letters out of here.